static/frontEndLayout.js

```diff
@@ -351,6 +358,105 @@ var buyPlot = function() {
     }
 }
 
+var createSeedMenu = function() {
+    for (var id in RECIPE_CONFIG) {
+        if (RECIPE_CONFIG.hasOwnProperty(id) && document.getElementById("row" + id) == null) {
+            var container = document.getElementById("seedCosts");
+                var tr = document.createElement("TR");
+                tr.id = "row" + id;
+            container.appendChild(tr);
+
+                var td1 = document.createElement("TD");
+                    var img1 = document.createElement("IMG");
+                    img1.src = GAME_CONFIG.seeds[id].imageSmall;
+                td1.appendChild(img1);
+
+                var td2 = document.createElement("TD");
+                    var div2 = document.createElement("DIV");
+                    div2.style.textAlign = 'left'
+                    div2.innerHTML = GAME_CONFIG.seeds[id].name;
+                td2.appendChild(div2);
+
+                var td3 = document.createElement("TD");
+                    var div3 = document.createElement("DIV");
+                    div3.id = "sell" + id;
+                    div3.classList.add('border');
+                    div3.addEventListener('click',createListener(sell,id));
+                td3.appendChild(div3);
+
+                var td4 = document.createElement("TD");
+                    var div4 = document.createElement("DIV");
+                    div4.id = "owned" + id;
+                td4.appendChild(div4);
+
+                var td5 = document.createElement("TD");
+                    var div5 = document.createElement("DIV");
+                    div5.classList.add('border');
+                    div5.id = 'buy' + id;
+                    div5.addEventListener('click',createListener(buy,id));
+                        var div5a = document.createElement("DIV");
+                            div5a.id = "buy" + id + "_label";
+                    div5.appendChild(div5a);
+                td5.appendChild(div5);
+
+                var td6 = document.createElement("TD");
+                    var img6 = document.createElement("IMG");
+                    img6.id = "resource" + id;
+                    img6.style.height = '20px';
+                td6.appendChild(img6);
+
+                var td7 = document.createElement("TD");
+                    var div7 = document.createElement("DIV");
+                    div7.id = "resourceCost" + id;
+                td7.appendChild(div7);
+
+            tr.appendChild(td1);
+            tr.appendChild(td2);
+            tr.appendChild(td3);
+            tr.appendChild(td4);
+            tr.appendChild(td5);
+            tr.appendChild(td6);
+            tr.appendChild(td7);
+
```

Comment:
These last document.getElementByIds belong in the above code.  For instance, td6 should check the varying costs and append an IMG and appropriate text nodes.

Also, by having one `"resource" + id` element, you're not going to be able to display a recipe that uses two or more non-cash resources.